docs/storage.md

```diff
@@ -58,7 +58,7 @@ Current object storage client implementations:
 | [Google Cloud Storage](./storage.md#gcs) | Stable  (production usage)             | yes       | @bwplotka   |
 | [AWS/S3](./storage.md#s3) | Stable  (production usage)               | yes        | @bwplotka          |
 | [Azure Storage Account](./storage.md#azure) | Stable  (production usage) | no       | @vglafirov   |
-| [OpenStack Swift](./storage.md#openstack-swift)      | Beta  (working PoCs, testing usage)               | no        | @sudhi-vm   |
+| [OpenStack Swift](./storage.md#openstack-swift)      | Beta  (working PoCs, testing usage)               | yes       | @sudhi-vm   |
```

Comment:
I guess we can actually put @FUSAKLA  here? (: :hugs:

Reply:
I'm already changing the maintainer in the https://github.com/thanos-io/thanos/pull/2732 but feel free to change it here, I'll be rebasing it nonetheless :)

Reply:
Yes, so let's merge this and then in https://github.com/thanos-io/thanos/pull/2732 let's add @FUSAKLA as the maintainer :+1: